### PR TITLE
Minor changes & fixes

### PR DESCRIPTION
**Pull request by boswelja**
* Make gradle libraries easier to upgrade
* Tint splash screen progress spinner
* Make notification channel preference go straight to settings, rather than presenting us with a list of channels in a dialog
* Removed what appears to be a useless class (TransientBottom) and override shouldAnimate() in the Lunchbar class
* Set ContentDescription on the splash screen logo

Next, I would like to take a look at the app immediately closing if Andromeda is not running, and show a dialog rather than just a notification. Also, maybe make UI blurring optional? It looks ugly IMO

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -113,29 +113,35 @@ android {
...
+    firebaseVersion = '11.2.2'
+    glideVersion = '4.1.1'
+}
 dependencies {
     implementation fileTree(include: ['*.jar'], dir: 'libs')
     // Leak Checks and Test Cases
-    debugImplementation 'com.squareup.leakcanary:leakcanary-android:1.5.1'
-    releaseImplementation 'com.squareup.leakcanary:leakcanary-android:1.5.1'
-    testImplementation 'com.squareup.leakcanary:leakcanary-android-no-op:1.5.1'
+    debugImplementation 'com.squareup.leakcanary:leakcanary-android:' + leakCanaryVersion
```
Comment (andrzejressel) [inline app/build.gradle]: I think "com.squareup.leakcanary:leakcanary-android:$leakCanaryVersion" will look much better.

Comment (boswelja) [reply app/build.gradle]: OK, changing it now

**Review by msfjarvis — COMMENTED** [review]
Not big on the changed indentations in layouts, looks fine otherwise. Will let Nick do the final code review and merge this into dev.

**Comment by nicholaschum** [comment]
looks good guys! merged

**Comment by boswelja** [comment]
@nicholaschum @MSF-Jarvis what do you think about removing blurring from views? Or even making it optional?

**Comment by msfjarvis** [comment]
@boswelja We leave it as is, no toggles.

**Comment by vaaesh** [comment]
@MSF-Jarvis *Anirudh approves*

**Comment by anirudhgupta109** [comment]
@vaaesh 👍

**Comment by vaaesh** [comment]
@anirudhgupta109 Stalkin' Horshe on GitHub gdi

**Comment by boswelja** [comment]
Just gonna link to this issue I created that apparently nobody has commented on
#125

**Comment by nicholaschum** [comment]
@boswelja Blur is being decided on but no toggles.

**Pull request merged**